.github/scripts/publish_env.sh

```diff
@@ -0,0 +1,35 @@
+# Parse the release tag
+
+# strip leading 'release/'
+TAG="${GITHUB_REF_NAME#release/}"
+
+# strip leading v
+TAG_VALUE="${TAG#v}"
+
+# strip trailing -dry
+VERSION="${TAG_VALUE%-dry}"
+
+# detect valid semver
+VALID_VERSION=$(npx -y semver-parser-cli@0.2.0 "${VERSION}" --field matches)
+if [ "${VALID_VERSION}" != "true" ]; then
+  exit 1
+fi
+
+# Detect dry run mode
+DRY_RUN=0
+if [ "${TAG_VALUE}" != "${VERSION}" ]; then
+    DRY_RUN=1
+fi
+
+# publish tag ('alpha', 'beta', etc.) is used to tag the release
+PUBLISH_TAG="$(npx -y semver-parser-cli@0.2.0 "${VERSION}" --field preid)"
+if [ "${PUBLISH_TAG}" == "undefined" ]; then
+  PUBLISH_TAG=latest
+fi
+
+echo "API_CLIENT_NAME=Rust"
+echo "PROJECT_NAME=fastly-rust"
```

Comment:
Does this need to align with the crate name? https://crates.io/crates/fastly-api

Reply:
This is only for output, and I've been using the project name for this.  for example, fastly-js is the `fastly` package, but I am using fastly-js here.